docs/plugins/tgeo.rst

```diff
@@ -1,6 +1,57 @@
 TGeo plugin
-===========
+***********
```

Comment:
```suggestion
===========
```

I think, this must stay `=====` for consistency with all other document sections. Otherwise, the TGeo plugin would be treated as a chapter and not as a section by Sphinx.

Reply:
Done.